app/Components/Sudoku Board/Components/Cell.tsx

```diff
@@ -325,27 +334,27 @@ const styles = (cellSize?: number, themeColor?: any) =>
     conflict: {
       // styles for cells with conflict prop
       color: "#000000",
-      backgroundColor: "#FFC3BF",
+      backgroundColor: NOT_SELECTED_CONFLICT_COLOR,
     },
     peer: {
       // styles for cells with isPeer prop
       color: "#000000",
-      backgroundColor: "#C5DDF4",
+      backgroundColor: PEER_SELECTED_COLOR,
     },
     sameValue: {
       // styles for cells with sameValue prop
       color: "#000000",
-      backgroundColor: "#c8dcc4",
+      backgroundColor: VALUE_SELECTED_COLOR,
```

Comment:
These cells are not selected, can we maybe refactor this to 'TWIN_COLOR'? 
I am trying to figure out if there is an official name for this but can't seem to find anything.

Reply:
Twin usually refers to exactly two of something but there could be more than two. What about CORRESPONDING_COLOR?

Reply:
maybe IDENTICAL_VALUE_COLOR?

Reply:
Sounds good